data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,612 @@ mission "Heliarchs Buy Jump Drive Mission"
+		conversation
+			`As you enter <system>, Anna walks out of her bunk and finds a spot on the side of the bridge, peering into the empty void. "What a massive ringworld!" she exclaims.`
+			choice
+				`	"It's enormous. As you know, they stole it from the Quarg roughly seven thousand years ago."`
+				`	(Say nothing.)`
+			`	"This is going to be a great source of inspiration for my next novel," says Anna, still in awe. She pulls out a small notebook and begins busily writing in it.`
+	on complete
+		conversation
+			`Anna is amazed as you touch down on the ringworld. As the two of you open the hatch, you see a Heliarch agent in front of you.`
+			`	"Greetings, humans. For you, what can I do today?" asks the Heliarch. You respond that you would like to register Anna with the Heliarchy. After reaching the registration office, Anna enters it, coming out after fifteen minutes. "They finally got all the paperwork done. I'm off to the spaceport to get some food. Meet me there to discuss our next move." The Heliarch follows her to the cafeteria area.`
+
+
+mission "Coalition: Struggling Author 3"
+	name "Back to <planet>"
+	description "Now that Anna Fangvin has completed the registration process, take her and the Heliarch agent to get a translation device on <destination>."
+	source "Ring of Friendship"
+	destination "Ki Patek Ka"
+	passengers 2
+	blocked "You do not have enough space for Anna and the Heliarch agent. Return here when you have two bunks free."
+	to offer
+		has "Coalition: Struggling Author 2: done"
+	on offer
+		conversation
+			`Going through the spaceport looking for Anna, you finally find her at a restaurant, eating a longcow steak and typing something on her laptop. Beside her sits a Heliarch Saryd, who points at something on the screen. She sees you and motions for you to take a seat.`
+			`	"This meat is so delicious! After months of instant noodles and takeout back home, it finally feels good to eat something that's actually healthy," she exclaims, closing the computer.`
+			choice
+				`	"How good is it?"`
+					goto good
+				`	"What did you usually eat back in human space?"`
+			`	"My diet wasn't exactly the best. All I had was carbs and takeout food; I never could find time in my schedule to cook at home. I was working hard on 'Beyond the Hyperdrive' at the time. This food is better than anything I could ever find on Zug.`
+				goto published
+			label good
+			`	"It's very good. Probably better than anything I've had back on Zug. Now, I'd like to get my book published here."`
+				goto company
+			label published
+			`	"On another note, I want to get my book published here."`
+			label company
+			`	"Have you found a publishing company yet?" you ask.`
+			`	"No," Anna replies. "But this Heliarch has been very helpful in the search for an adequate publishing company. Had it not been for the agent, it would have been extremely difficult for me to communicate with the restaurant staff, as they do not have a translation box."`
```

Comment:
```suggestion
			`	"No," Anna replies. "But this Heliarch has been a huge help in finding a suitable publisher. Without this agent, it would've been nearly impossible for me to communicate with the restaurant staff, since they don't have a translation box."`
```